library/include/hipblaslt-ext-op.h

```diff
@@ -165,8 +174,11 @@ HIPBLASLT_EXPORT hipblasStatus_t hipblasltExtLayerNorm(hipDataType datatype,
  */
 HIPBLASLT_EXPORT hipblasStatus_t hipblasltExtAMax(const hipDataType datatype,
```

Comment:
should add another function overloading but rather than directly change original API.